index.js

```diff
@@ -35,12 +37,13 @@ client.on('message', (message) => {
     const roll = roller(dice, targetNumber, neededSuccesses, !!hasFocus ? focus: 0);
 
     const rollsOutput = roll.rolls.map(result => {
-       return result.focused ? '**' + result.value + '**': result.value;
+        return result.focused ? result.value + `*(+${result.focused})*`: result.value;
```

Comment:
The rest is just some formatting to make it easier for vim users to move around (new lines and a shortening of the output line). This is the actual change.